spec/takeaway_spec.rb

```diff
@@ -4,14 +4,15 @@
 describe Takeaway do
   it 'shows the list of dishes with prices' do
     food_person = Menu.new
-    food_person.list
+    food_person.food_list
     expect(subject.show_menu).to eq food_person.food_list
   end
 
   describe '#select' do
-    let(:food){ double :food }
+    let(:food) { double :food }
+    let(:quantity) { double :quantity }
     it 'puts the selected dishes to the cart' do
-      expect(subject.select(food)).to eq "#{food} added to your cart!"
+      expect(subject.select(food, quantity)).to eq "#{food} added to your cart!"
```

Comment:
nicely written test! would read even better if the subject was explicitly defined